sharding/nodesCoordinator/hashValidatorShuffler.go

```diff
@@ -69,27 +69,25 @@ func NewHashValidatorsShuffler(args *NodesShufflerArgs) (*randHashShuffler, erro
 	if args == nil {
 		return nil, ErrNilNodeShufflerArguments
 	}
+	if check.IfNil(args.EnableEpochsHandler) {
+		return nil, ErrNilEnableEpochsHandler
+	}
 
 	var configs []config.MaxNodesChangeConfig
 
 	log.Debug("hashValidatorShuffler: enable epoch for max nodes change", "epoch", args.MaxNodesEnableConfig)
```

Comment:
forgotten flag?

Reply:
`MaxNodesEnableConfig` is a config, not a specific epoch from `enableEpochs.toml`